submariner-operator/issue-149-node-placement.md

```diff
@@ -0,0 +1,96 @@
+# Node Placement
+
+[Add support for nodeSelector and toleration scheduling for submariner pods](https://github.com/submariner-io/enhancements/issues/149)
+
+## Summary
+
+Submariner deploys many different agents as part of its core deployment. Gateway, Globalnet Agent and
+Metrics Proxy are deployed on designated gateway nodes, while Route Aents are deployed on all nodes. There is no way
+for users to control or limit where rest of the agent pods are deployed. Some customers want to run these agents on
+specific nodes for billing or networking reasons. Users can't add tolerations directly to these agents coz they'll be
```

Comment:
```suggestion
specific nodes for billing or networking reasons. Users can't add tolerations directly to these agents because they'll be
```

nit :)